sn_client/src/api.rs

```diff
@@ -67,7 +67,7 @@ impl Client {
     ///
     /// Optionally specify the duration for the connection timeout.
     ///
-    /// Defaults to 180 seconds.
+    /// Defaults to [`CONNECTION_TIMEOUT`].
```

Comment:
For this to properly link, it has to be exported by `lib.rs` too, with e.g.
```
pub use api::CONNECTION_TIMEOUT;
```
Or making the full module public by `pub mod api;`

Reply:
You can check quickly if the doc is proper by compiling the docs: `cargo doc --package=sn_client --open --no-deps`

Reply:
I've done it with 
```rust
pub(crate) const CONNECTION_TIMEOUT: Duration = Duration::from_secs(30);
```